### ISSUE TITLE
A built-in way to download themes and put them in the correct place

### ISSUE THREAD

**Issue by nul800sebastiaan** [umbraco/Umbraco.Forms.Issues#1162]
I often see people struggling to customize the theme they're using, very much unaware of the option to download the themes or why this is needed. 

Would be nice to have a dashboard that helps to download the theme for their current version and unzip it in the right place. Additionally it should have some documentation explaining that they should add the files to source control and (if needed) to their csproj to make sure it deploys. 

Maybe this dashboard needs to be only available when the site is not in the runtime mode `Production`.

**Comment by AndyButland** [comment]
Yes, will give this some thought - the question does come up a few times.

Just for reference - and in case anyone else lands on this - the links for download and details are documented [here](https://docs.umbraco.com/umbraco-forms/developer/themes).

One potential issue is that you don't necessarily need all the theme files - if you are just updating a small part, you can just update one or a few files and keep using the out of the box ones for other files (which is useful if any fixes to those files are made, as you'll get them automatically).  And so downloading a copy of all might not be be best.

**Comment by nul800sebastiaan** [comment]
That's fair, but having some guidance in the UI to inform people about would be good. Even if we can't decide for them exactly which files they would want, we can say something like: 

> this will download all files, don't forget to add them to your source control and feel free delete the ones you don't want or need to change.
